app/controllers/shopify_app/callback_controller.rb

```diff
@@ -129,7 +129,7 @@ def token
     end
 
     def access_scopes
-      return unless auth_hash['extra']['scope']
+      return unless auth_hash.dig('extra', 'scope')
```

Comment:
Couldn't both these lines be replaced by:

`auth_hash.dig('extra', 'scope')`

?

Reply:
Yeah, that's better. Updated.